OpenMTRDemo/Forms/ExpandedImageDialog.cs

```diff
@@ -3,46 +3,67 @@
 using OpenCvSharp;
 using OpenMTRDemo.Forms;
 
-namespace OpenMTRDemo
+namespace OpenMTRDemo.Forms
 {
-    public partial class MainForm : BaseForm
+    public partial class ExpandedImageDialog : BaseForm
```

Comment:
The name should reflect what it is "ExpandedImageForm." I'm not sure why you went off naming convention here.  Also all of the filtering elements are disabled. Was this tested?

Reply:
Yes this was tested, and this works.
The reason for going off the naming convention was to resolve a problem with Travis CI, see the log from commit c363929 and prior.

Reply:
Oh, the filtering elements... You know what I wasn't really paying a lot of attention to those, I'll check on it.